### ISSUE TITLE
 How should the hyperparameters be appropriately set for training on publicly available datasets such as DUTS-TR, and for ECSSD for significance detection?

### ISSUE THREAD

**Issue by tongxiaozhong14** [milesial/Pytorch-UNet#222]
Dear author, thank you very much for sharing the dataset, but I am training on public datasets such as DUTS-TR (10,553 images and mask (target is white, rest is black, num_classes=1)) and other datasets such as ECSSD for significance detection, and the training results are very unsatisfactory, almost nothing can be detected, how should I set the hyperparameters such as LR, epochs, loss and optimizer etc. to ensure that the training results are more satisfactory?
I hope to get a response soon.

**Comment by milesial** [comment]
Typically you would want to search hyperparameters with a grid or Bayesian search.
